src/Uno.UI/Helpers/PlatformRuntimeHelper.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Uno.UI.Helpers;
+
+public static class PlatformRuntimeHelper
+{
+	internal static UnoRuntimePlatform SkiaPlatform { get; set; } = UnoRuntimePlatform.Skia;
+
+	public static UnoRuntimePlatform Current => GetPlatform();
+
+	private static UnoRuntimePlatform GetPlatform() =>
```

Comment:
Please add a runtime test that gets the value from `GetPlatform()` at runtime and checks that the result is not `Unknown` - this will future proof us against potential new platforms being added and us forgetting to update this code path